wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -185,6 +190,7 @@ pub enum WalletCommand {
         number_of_decimals: u8,
```

Comment:
`amount_to_issue` on line 189 is now obsolete

Reply:
yep, forgot to remove that